mxnet-bot/LabelBotFullFunctionality/LabelBot.py

```diff
@@ -253,7 +253,7 @@ def parse_webhook_data(self, event):
             if "@mxnet-label-bot" in payload["comment"]["body"]:
                 labels += self._tokenize(payload["comment"]["body"])
                 if not labels:
-                    logging.error("Message typed by user: {}".format(payload["comment"]["body"]))
+                    logging.error('Message typed by user: {}'.format(payload["comment"]["body"]))
```

Comment:
nit: the "modern" way to do logging is the following notion:

``logging.error('Message typed by user: %s', payload["comment"]["body"])``

The advantage of that approach is that it lazy loads the parameters. This increases the performance since parameters are only evaluated if they're actually used. Also, it's a bit nicer to read IMO :)